bedrock/base/templates/includes/banners/fundraiser.html

```diff
@@ -44,7 +44,7 @@
   {% set banner_button = 'Donate' %}
 {% endif %}
 
-{% set banner_link = 'https://mozillafoundation.org/?form=gt25-moco-banner ' %}
+{% set banner_link = 'https://www.mozillafoundation.org/?form=gt25-moco-banner ' %}
```

Comment:
Not a regression from this work, but fixing while I see it

```suggestion
{% set banner_link = 'https://www.mozillafoundation.org/?form=gt25-moco-banner' %}
```